lib/api_resources/root.js

```diff
@@ -225,6 +225,10 @@ RootResource.prototype._renderRoot = function(env, next) {
         title: this.server._name,
         rel: [rels.server],
         href: env.helpers.url.path('/servers/' + encodeURI(this.server.id) )
+      },
+      {
+        rel: [rels.events],
+        href: env.helpers.url.path('/events').replace(/^http/, 'ws') 
```

Comment:
Talked with @AdamMagaluk thinking that we actually remove this link for SPDY responses entirely. Currently this feature is only really going to work over websockets.